handler_test.go

```diff
@@ -0,0 +1,126 @@
+package rerpc_test
+
+import (
+	"bytes"
+	"context"
+	"encoding/base64"
+	"encoding/json"
+	"io"
+	"net/http"
+	"net/http/httptest"
+	"strconv"
+	"strings"
+	"testing"
+
+	"google.golang.org/protobuf/proto"
+
+	"github.com/rerpc/rerpc"
+	"github.com/rerpc/rerpc/internal/assert"
+	pingpb "github.com/rerpc/rerpc/internal/ping/v1test"
+	statuspb "github.com/rerpc/rerpc/internal/status/v1"
+	"github.com/rerpc/rerpc/internal/twirp"
+)
+
+func TestHandler(t *testing.T) {
+	const readMaxBytes = 32
+	ping := rerpc.NewHandler(
+		"internal.ping.v1test.PingService.Ping", // fully-qualified protobuf method
+		"internal.ping.v1test.PingService",      // fully-qualified protobuf service
+		"internal.ping.v1test",                  // fully-qualified protobuf package
+		rerpc.Func(func(ctx context.Context, req proto.Message) (proto.Message, error) {
+			return &pingpb.PingResponse{}, nil
+		}),
+		rerpc.ReadMaxBytes(readMaxBytes),
+	)
```

Comment:
Since we're testing through the HTTP layer, I think we can use the generated code instead of calling `NewHandler` directly.

Reply:
This makes a lot of sense.